src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -275,7 +309,7 @@ public void findLogs(@NotNull World world)
      * Checks if the tree has been planted from more than 1 saplings.
      * Meaning that more than 1 log is on the lowest level.
      *
-     * @param world  The world where the tree is in
+     * @param world  The world where the tree is in.
      * @param yLevel The base y.
      */
     public void fillTreeStumps(@NotNull World world, int yLevel)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "world". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)